src/commands/aicommits.ts

```diff
@@ -56,19 +57,19 @@ export default async (
 	let message: string;
 	if (messages.length === 1) {
 		[message] = messages;
-		const confirmed = await confirm({
-			message: `Use this commit message?\n\n   ${message}\n`,
-		});
+		const confirmed = noninteractive ? true : await confirm({ message: `Use this commit message?\n\n   ${message}\n` });
 
 		if (!confirmed || isCancel(confirmed)) {
 			outro('Commit cancelled');
 			return;
 		}
 	} else {
-		const selected = await select({
-			message: `Pick a commit message to use: ${dim('(Ctrl+c to exit)')}`,
-			options: messages.map(value => ({ label: value, value })),
-		});
+		const selected = noninteractive
+			? messages[1]
```

Comment:
```suggestion
			? messages[0]
```

Why not pick the first message?

Reply:
lol I thought I was already doing it but totally missed it that I put "1" instead of "0" 😆 thanks!